File: Omnibus_and/app/src/main/java/com/example/asuper/omnibus/Klasy/Zestaw.java
package com.example.asuper.omnibus.Klasy;

import java.io.Serializable;
import java.util.ArrayList;

/**
 * Created by super on 15.01.2017.
 */

public class Zestaw implements Serializable {
    public  Pytanie pyt;// { get; set; }
    public ArrayList<Odpowiedz> odp;// { get; set; }

    public Zestaw(Pytanie pyt, ArrayList<Odpowiedz> odp) {
        this.pyt = pyt;
        this.odp = odp;
    }

    public Pytanie getPyt() {
        return pyt;
    }

    public void setPyt(Pytanie pyt) {
        this.pyt = pyt;
    }

    public ArrayList<Odpowiedz> getOdp() {
        return odp;
    }

    public void setOdp(ArrayList<Odpowiedz> odp) {
        this.odp = odp;
    }
}

File: WebService/WebService/Models/Zestaw.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebService.Models
{
    public class Zestaw
    {
      public  Pytanie pyt { get; set; }
        public IQueryable<Odpowiedz> odp { get; set; }
    }
}